src/init.cpp

```diff
@@ -752,13 +753,19 @@ void InitParameterInteraction()
 
     // disable walletbroadcast and whitelistalwaysrelay in blocksonly mode
```

Comment:
Valid point, although how old can these conf files be, given that `whitelistalwaysrelay` was introduced in #6993 only two months ago, so isn't in any releases.